spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/main/java/org/springframework/cloud/kubernetes/configuration/watcher/ConfigurationWatcherConfigurationProperties.java

```diff
@@ -28,6 +28,16 @@
 @ConfigurationProperties("spring.cloud.kubernetes.configuration.watcher")
 public class ConfigurationWatcherConfigurationProperties {
 
+	/**
```

Comment:
we don't really need getters and setters for these two properties. I don't strictly need this changes in this PR, but in the next one (where we will be adding two more properties), this will become much more cleaner. imo, at least